Runtime/Game/LootLockerSDKManager.cs

```diff
@@ -3932,6 +3932,23 @@ public static void ComputeAndLockDropTable(int tableInstanceId, Action<LootLocke
             LootLockerAPIManager.ComputeAndLockDropTable(tableInstanceId, onComplete, AddAssetDetails, tag);
         }
 
+        /// <summary>
+        /// Lock a drop table and return information about the assets that were computed.
+        /// </summary>
+        /// <param name="tableInstanceId">Asset instance ID of the drop table to compute</param>
+        /// <param name="AddAssetDetails">If true, return additional information about the asset</param>
```

Comment:
I know it's not in your code, but could you add to the documentation of this for the variant that the parameters in question are optional? Like:
```<param name="AddAssetDetails">Optional: If true, return additional information about the asset</param>```